### ISSUE TITLE
Low Priority: Fix Indentation of inserted pragmas

### ISSUE THREAD

**Issue by lukasrothenberger** [Devincf/DiscoPoP-for-VSCode#1]
Hi @Devincf ,

It would be nice if the indentation of an inserted pragma and the "targeted" statement would be equal.
You can see an example in the attached image.

Best,
Lukas

![Bildschirmfoto vom 2021-08-24 09-59-55](https://user-images.githubusercontent.com/26957074/130579631-296213fe-d60c-484b-817a-d3ed1440fee7.png)

**Comment by Devincf** [comment]
Hey,
this should be a quick fix, I'll implement it when possible and push out a new version -- probably only after the thesis has been graded though as I don't want to change the code before that.
